vimeo-networking/src/main/java/com/vimeo/networking/model/FeedItem.java

```diff
@@ -46,44 +49,89 @@
         NONE
     }
 
-    @GsonAdapterKey("uri")
-    public String uri;
-    @GsonAdapterKey("clip")
-    public Video clip;
-    @GsonAdapterKey("type")
-    public String type;
-    @GsonAdapterKey("time")
-    public Date time;
-    @GsonAdapterKey("user")
-    public User user;   // from like type
-    @GsonAdapterKey("channel")
-    public Channel channel; // from channel type
-    @GsonAdapterKey("category")
-    public Category category;
-    @GsonAdapterKey("tag")
-    public Tag tag;
-    @GsonAdapterKey("group")
-    public Group group;
-    @GsonAdapterKey("metadata")
-    public Metadata metadata;
+    @SerializedName("uri")
+    protected String mUri;
+
+    @SerializedName("clip")
+    protected Video mClip;
```

Comment:
I've added the setter, but I envision a second PR that adds all necessary setters as I'm sure there's a lot of places in vimeo-kit and Vimeo-android where we need to set field value or need to add additional getters